packages/vats/test/bootstrapTests/supports.js

```diff
@@ -488,4 +497,94 @@ export const makeSwingsetTestKit = async (
     timer,
   };
 };
+
 /** @typedef {Awaited<ReturnType<typeof makeSwingsetTestKit>>} SwingsetTestKit */
+
+export const makeTestContext = async t => {
```

Comment:
it's only the context for `test-vats-restart`. why move this from that module to this shared one?

Reply:
I thought there would be more sharing.  I moved it back.